CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required (VERSION 2.6)
 project (tunnler CXX)
 
+if(UNIX)
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=gnu++14")
+endif()
```

Comment:
Only my stupidity and my inability of writing cmake. I will just look what citra uses and add it here